### DISCUSSION TITLE
Colab plots not working

### DISCUSSION THREAD

**Discussion by drob-xx** [pycaret/pycaret#2003] [Q&A] [answered]
I'm just working through the basic NLP tutorial on Colab and I can't get plot_model() to output a plot. I've tried both using and omitting enable_colab(). Any ideas?

**Comment by moezali1** [comment]
@drob-xx 

I was able to make it work by updating plotly version by `pip install --upgrade plotly` . Also remember to do execute this code:

```
from pycaret.utils import enable_colab
enable_colab()
```
![image](https://user-images.githubusercontent.com/54699234/147797811-498e2375-53ce-4907-b5da-cb2b815015cd.png)

**Reply by drob-xx** [reply]
Thanks for taking the time!  I've been running enable_colab() and getting the success message. I just did the --upgrade of plotly (which is now 5.5.0 up from 4.4.1, but now get an error: TypeError: load() missing 1 required positional argument: 'Loader'.  What version are you running?

**Reply by drob-xx** [reply]
Still interested in your version of plotly. However, someone else (same version Chrome and Windows) was able to load and run everything just fine. I re-booted, removed cookies (I had previously turned off plug-ins) but still nothing. I installed Firefox and it worked no problem

**Comment by drob-xx** [comment, marked as answer]
Worked with Firefox.
